contrib/inventory/cloudforms.ini

```diff
@@ -27,6 +27,9 @@ clean_group_keys = True
 # Explode tags into nested groups / subgroups
 nest_tags = False
 
+# If set, ensure host name are suffixed with this value
+# suffix = example.org
```

Comment:
If people use example.org, the '.' is missing, shouldn't it be noted somewhere ?

Reply:
@mscherer Updated value and expanded description